util/log.go

```diff
@@ -19,24 +21,30 @@ func InitLog(logLevel string, logPath string) error {
 		return err
 	}
 
+	logPath = "/tmp/netbird/netbird/netbird.log"
 	if logPath != "" && logPath != "console" {
```

Comment:
I recommend to move this scope into a new function. And always good idea to avoid the deep if-else statements.
I. e.
```
if logPath == "" {
  return
}
if logPath == "console" {
  return
}

if !canWrite {
 return
}
...

```

Reply:
I point to the 25. line. Sry